pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -390,6 +395,17 @@ var supportedReinvocationPolicies = sets.NewString(
 	string(admissionregistration.IfNeededReinvocationPolicy),
 )
 
+var supportedValidationPolicyReason = sets.NewString(
+	string(metav1.StatusReasonUnauthorized),
+	string(metav1.StatusReasonForbidden),
+	string(metav1.StatusReasonNotFound),
```

Comment:
returning StatusReasonNotFound / 404 to a client on a write/delete request is going to be misleading, since clients will assume the object no longer exists and may drop if from their processing queues. I think we should drop this one